apis/fluent-en_GB/atrium-api-fluent-en_GB-common/src/main/kotlin/ch/tutteli/atrium/api/fluent/en_GB/mapEntryAssertions.kt

```diff
@@ -31,6 +33,8 @@ val <K, T : Map.Entry<K, *>> Expect<T>.key: Expect<K>
  * returns an [Expect] for the current subject of `this` expectation.
  *
  * @return an [Expect] for the subject of `this` expectation.
+ *
+ * @sample ch.tutteli.atrium.api.fluent.en_GB.samples.MapEntryAssertionSamples.key
```

Comment:
Can you please also add an example for `val ... key` and `val... value` (name it function keyFeature() and valueFeature() in MapEntryAssertionSamples.kt